_src/static/sass/utilities/_variables.scss

```diff
@@ -233,6 +230,17 @@ $font-weights: (
     x-bold:         800
 );
 
+// typography: letter-spacing
+$letter-spacing: (
+    normal:         0,
+    tightest:       -3,
+    tighter:        -2,
+    tight:          -.095rem,
+    loose:          -.075rem,
+    looser:         -.025rem,
+    loosest:        3
+);
+
```

Comment:
I'd prefer if we use the same naming convention we used in other scales/maps (see https://github.com/edx/ux-pattern-library/blob/master/_src/static/sass/utilities/_variables.scss#L215).